scripts/create-hsm-wallet.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+const { hsm } = require('@dinamonetworks/hsm-dinamo');
+const config = require('../src/config');
+
+/**
+ * Creates a new xPriv key on the HSM
+ */
```

Comment:
suggestion: Add typing on the docstring.

If the dinamo lib has types we could use them.

Reply:
Done on [dc67315](https://github.com/HathorNetwork/hathor-wallet-headless/pull/357/commits/dc67315eef6aabac43c1aa7d6afb1230dabb60c5).

Could you please check the javascript docsting parser you currently use can identify this `@type {Hsm}` on the `createXprivKey`?